roles/docker/templates/policy.json.j2

```diff
@@ -0,0 +1,3 @@
+{"name":"policy_1","users":["core", "root"],"actions":[""]}
+{"name":"policy_2","users":["service_account"],"actions":["container_logs","container_top"]}
+{"name":"policy_3","users":["kube"],"actions":["container"], "readonly":true }
```

Comment:
kube is the service account we have at the moment for the default namespace and it can only read containers